docs/datatypes/io.md

```diff
@@ -164,38 +164,30 @@ val composite = action.ensuring(cleanupAction)
 ### A full working example on using brackets
 ```scala mdoc:silent
 
-import scalaz.zio.{ App, DefaultRuntime, Task, UIO }
+import scalaz.zio.{App, Task, UIO}
 
-import java.io.{ File, FileInputStream }
+import java.io.{File, FileInputStream }
 import java.nio.charset.StandardCharsets
 
 object Main extends App {
 
-  val rt = new DefaultRuntime {}
-
   // run my bracket
   def run(args: List[String]) =
-    mybracket.orDie.map(_ => 0)
+    mybracket.orDie.const(-1) // return -1 on fail
 
   def closeStream(is: FileInputStream) =
-    UIO.effectTotal(is.close())
+    UIO(is.close())
 
   def convertBytes(is: FileInputStream) =
-    Task.effect(println(new String(is.readAllBytes(), StandardCharsets.UTF_8)))
+    Task.effect(println(new String(is.readAllBytes(), StandardCharsets.UTF_8))) // Java 11+
```

Comment:
We don't build yet on Java 11, so this probably won't typecheck.

Reply:
Fixed

This may be an issue. I run my code for Java 11 seamlessly. The very same code fails to compile `is.readAllBytes()` on Java8. What to do? I don't know how to refactor `readAllBytes`